### ISSUE TITLE
site reconnect error message truncated

### ISSUE THREAD

**Issue by nosilver4u** [mainwp/mainwp#603]
When a site gets disconnected, and one tries to reconnect it and the process fails, an error is shown. But this error message is incomplete and not helpful. I've already successfully fixed my site, so I don't recall the exact wording, but it's something like "An error has occurred, check the" And that's where it ends. It doesn't tell you what to check or where to look.
I had this same issue a month or so ago, and discovered a workaround, which is to remove the site entirely, and try re-adding it. Then, if the issue persists, you can view the actual response received by the MainWP dashboard.

1. Add a child site
2. Do something that will break the connection to the child site, like changing the site URL (non-www to www) or adding a firewall that blocks connections from MainWP to the child site.
3. Attempt to reconnect the site
4. See error message, but it'll be incomplete.

**Expected behavior**
I expect to see something that will tell me what happened. I suspect the HTML response is supposed to be available for inspection, just like when you're adding a fresh site and the connection fails.

**Comment by bogdan-mainwp** [comment]
Hi @nosilver4u,

Thanks for reporting the problem.

Can you please share a screenshot that will help us understand the problem better?

**Comment by nosilver4u** [comment]
Well, unfortunately I can't do a screenshot of the original problem, because now it doesn't even display an error message. I've got a site that's disconnected currently, and no idea why. Well, I have some idea why, because it's the same site as before. But it doesn't display any errors, just says the site is disconnected.
I've got a screenshot of the disconnect message, but not sure that's at all helpful.
<img width="670" alt="Screenshot 2023-04-06 at 6 33 55 AM" src="https://user-images.githubusercontent.com/6015678/230380128-a759ba49-df1e-4709-90e6-092ccb7d27f9.png">
